app/grandchallenge/evaluation/templates/evaluation/combinedleaderboard_detail.html

```diff
@@ -17,7 +17,7 @@
         <li class="breadcrumb-item"><a
                 href="{{ challenge.get_absolute_url }}">{% firstof challenge.title challenge.short_name %}</a></li>
         <li class="breadcrumb-item"><a
-                href="{% url 'evaluation:leaderboard' challenge_short_name=challenge.short_name slug=challenge.phase_set.first.slug %}">Leaderboards</a></li>
+                href="{% url 'evaluation:leaderboard' challenge_short_name=challenge.short_name slug=challenge.phase_set.first.slug %}">Combined Leaderboards</a></li>
```

Comment:
I think this should stay at Leaderboards because it links to the leaderboards not only the combined leaderboards.

Reply:
I checked and there is a template for leaderboards ([here](https://github.com/comic/grand-challenge.org/blob/6954da87bc61d47ae23f7c30469b3d8cf05629cf/app/grandchallenge/evaluation/templates/evaluation/leaderboard_detail.html)) other than combined leaderboards.

For example, in the leaderboard tab, both phase leaderboards and combined leaderboards will have the word "Leaderboards" in the breadcrumb and title, as in below:

![Screenshot 2024-10-18 at 13 41 47](https://github.com/user-attachments/assets/95c38753-634f-468a-a45e-762607c04681)

![Screenshot 2024-10-18 at 13 41 59](https://github.com/user-attachments/assets/d0a5f63d-b5b3-40a4-9813-c3eeeb9d331a)

While the "combined leaderboard" settings and delete confirmation views, they will have "Combined Leaderboards" in the breadcrumb and title, as in below:

![Screenshot 2024-10-18 at 13 42 06](https://github.com/user-attachments/assets/d135f2c3-bed1-4b78-9124-68a8e338e0d8)

Do I get it right?

Reply:
Yes, you are right, but I think that actually does not make sense. The breadcrumbs that read "Leaderboards" and those that read "Combined leaderboards" all lead to the same view (i.e. the first challenge leaderboard), so I think the "Combined Leaderboards" breadcrumb is misleading and should be replaced with just "Leaderboards" everywhere

Reply:
okay, I made it "Leaderboards" in the three templates for combinedleaderboard